src/coreclr/jit/llvm.cpp

```diff
@@ -592,6 +594,9 @@ bool Llvm::helperCallHasManagedCallingConvention(CorInfoHelpFunc helperFunc) con
         // Dead code.
         { FUNC(CORINFO_HELP_THROW_TYPE_NOT_SUPPORTED) },
 
+        // TODO-LLVM:
```

Comment:
```suggestion
        // Not used in NativeAOT.
```
Source: https://github.com/dotnet/runtime/blob/2ee941b5e33b68fc39d2c6c2a26f025e0d2efee6/src/coreclr/tools/aot/ILCompiler.RyuJit/JitInterface/CorInfoImpl.RyuJit.cs#L452.